### ISSUE TITLE
Client disconnected: Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead.

### ISSUE THREAD

**Issue by TerraXSoul** [nyueki/ComfyUI-RemoteCLIPLoader#3]
Trying to use with ltx2

I get this on the worker:
Client disconnected: Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead.

On the client:
LTX2_NAG
mat1 and mat2 shapes cannot be multiplied (77x384 and 3840x4096)
